Message component

Jap9 opened the pull request:
Fixed some pending issues (UI responsiveness on Smartphone, disallow user to send message if not fields are filled)

Jap9 committed:
Changed navbar component to use navbar HTML to show unRead messages

Jap9 committed:
validates that the message sender matches logged in user prior sending a message

Jap9 committed:
closes #63 Message should display an error if user didn't fill in all fields

Jap9 committed:
closes #56

Jap9 committed:
message lint fixes

zurfyx commented:
https://github.com/UdL-EPS-SoftArch/softarch-1617-client/pull/64/commits/40ab5304bcb84ecdc6112047b3732c3e57e5f384 to https://github.com/UdL-EPS-SoftArch/softarch-1617-client/pull/64/commits/4ca83c5d139675e63903a4052b1988f34ba337f9 are duplicated of https://github.com/UdL-EPS-SoftArch/softarch-1617-client/commit/22d38f095295fb0c593f91f6084aa756809aade9 to https://github.com/UdL-EPS-SoftArch/softarch-1617-client/commit/52d602cb0435b59d9f85565924dd63c57e4e80f8

zurfyx committed:
undo 	e15f743, optimized #63
sender should not be validated, since server shouldn't require it (it should pick logged in user).

zurfyx committed:
closes #62
